<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sensor_SEN0105</class>
 <widget class="QWidget" name="Sensor_SEN0105">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>710</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../image.qrc">
    <normaloff>:/image/SOU0105_Icon.jpg</normaloff>:/image/SOU0105_Icon.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #132630;</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>460</y>
     <width>431</width>
     <height>241</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QTabWidget::pane { 
border: 1px solid #C0C0C0;
}</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="elideMode">
    <enum>Qt::ElideNone</enum>
   </property>
   <widget class="QWidget" name="tab_Infor_2">
    <attribute name="title">
     <string>     Hiển thị     </string>
    </attribute>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>431</width>
       <height>211</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #132630;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLCDNumber" name="lcdNumber_Value">
      <property name="geometry">
       <rect>
        <x>8</x>
        <y>9</y>
        <width>411</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #132630;
color: rgb(0, 255, 187);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="smallDecimalPoint">
       <bool>false</bool>
      </property>
      <property name="digitCount">
       <number>7</number>
      </property>
      <property name="segmentStyle">
       <enum>QLCDNumber::Flat</enum>
      </property>
      <property name="value" stdset="0">
       <double>0.000000000000000</double>
      </property>
      <property name="intValue" stdset="0">
       <number>0</number>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_Unit">
      <property name="geometry">
       <rect>
        <x>41</x>
        <y>69</y>
        <width>377</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QComboBox {
    border: 1px solid gray;
	color: rgb(255, 255, 255);
}
QComboBox::drop-down {
    width:35px;
}</string>
      </property>
      <item>
       <property name="text">
        <string>Độ ồn (dB)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Tần số (Hz)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Điện áp (V - Voltage)</string>
       </property>
      </item>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_6">
      <property name="geometry">
       <rect>
        <x>8</x>
        <y>110</y>
        <width>128</width>
        <height>81</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Giá trị lớn nhất</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Giá trị trung bình</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_10">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Giá trị nhỏ nhất</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_7">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>110</y>
        <width>261</width>
        <height>81</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QLabel" name="label_ValueMax">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(170, 0, 0);</string>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_ValueAvg">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_ValueMin">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 170, 0);</string>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>8</x>
        <y>69</y>
        <width>32</width>
        <height>32</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../image.qrc">
        <normaloff>:/image/SOU0105_Icon.jpg</normaloff>:/image/SOU0105_Icon.jpg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>       Thiết bị       </string>
    </attribute>
    <widget class="QWidget" name="verticalLayoutWidget_8">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>231</width>
       <height>101</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <item>
       <widget class="QLabel" name="label_Name">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_Address">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_SerialNumber">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>91</width>
       <height>131</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_9">
      <item>
       <widget class="QLabel" name="label_13">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Tên thiết bị:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_14">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Địa chỉ:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_18">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Mã sê-ri:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_19">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Mô tả:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_10">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>110</y>
       <width>231</width>
       <height>61</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_10">
      <item>
       <widget class="QLabel" name="label_Description">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>        Cài đặt        </string>
    </attribute>
    <widget class="QSpinBox" name="spinBox_SoundSensitivity">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>110</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QSpinBox
{
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
}
QSpinBox::up-button
{
    min-width:40px;
    min-height: 20px;
}
QSpinBox::down-button
{
    min-width: 40px;
    min-height: 20px;
}
</string>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>4096</number>
     </property>
     <property name="value">
      <number>500</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_SoundRepeat">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>160</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QSpinBox
{
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
}
QSpinBox::up-button
{
    min-width:40px;
    min-height: 20px;
}
QSpinBox::down-button
{
    min-width: 40px;
    min-height: 20px;
}
</string>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="value">
      <number>10</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_30">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>120</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Độ nhạy âm thoa:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_32">
     <property name="geometry">
      <rect>
       <x>114</x>
       <y>170</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Độ lặp lại:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_61">
     <property name="geometry">
      <rect>
       <x>94</x>
       <y>30</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Độ phân giải:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_Display_Unit_Pre">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>101</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QSpinBox
{
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
}
QSpinBox::up-button
{
    min-width:40px;
    min-height: 20px;
}
QSpinBox::down-button
{
    min-width: 40px;
    min-height: 20px;
}
</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>3</number>
     </property>
     <property name="value">
      <number>3</number>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>411</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_Exit">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>650</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(225, 225, 225);</string>
   </property>
   <property name="text">
    <string>Thoát</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame_Chart">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>1260</width>
     <height>421</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame { 
border: 1px solid #C0C0C0;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_OpenData">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>500</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(225, 225, 225);</string>
   </property>
   <property name="text">
    <string>Mở bản ghi</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>470</y>
     <width>501</width>
     <height>231</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="pushButton_Start">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>461</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(225, 225, 225);
	color: rgb(0, 0, 0);
}</string>
    </property>
    <property name="text">
     <string>Bắt đầu</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_SelectSample">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>461</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox {
    border: 1px solid gray;
	color: rgb(255, 255, 255);
}
QComboBox::drop-down {
    width:35px;
}</string>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <item>
     <property name="text">
      <string>Đo mức âm thanh</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Đo sóng tới và sóng phản xạ</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Đo tần số âm thoa</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Đo điện áp (Vrms)</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_SaveData">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>550</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(225, 225, 225);</string>
   </property>
   <property name="text">
    <string>Ghi dữ liệu</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame_trackLine">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame { 
border: 1px solid #C0C0C0;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="pushButton_TrackLine_Yax">
    <property name="geometry">
     <rect>
      <x>382</x>
      <y>2</y>
      <width>27</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>tính hệ số góc y=ax</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../image.qrc">
      <normaloff>:/image/MFS0103_Track_yax.jpg</normaloff>:/image/MFS0103_Track_yax.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_TrackLine_Default">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>2</y>
      <width>27</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>trục tọa độ ảo</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../image.qrc">
      <normaloff>:/image/MFS0103_Track_D.jpg</normaloff>:/image/MFS0103_Track_D.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_TrackLine_Tooltip">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>2</y>
      <width>27</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>bắt điểm</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../image.qrc">
      <normaloff>:/image/MFS0103_Track_T.jpg</normaloff>:/image/MFS0103_Track_T.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_setChart">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>420</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame { 
border: 1px solid #C0C0C0;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QRadioButton" name="radioButton_Continuously">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>2</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Đồ thị liên tục</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Accumulation">
    <property name="geometry">
     <rect>
      <x>152</x>
      <y>2</y>
      <width>131</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Đồ thị tích lũy</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ClearChart">
    <property name="geometry">
     <rect>
      <x>298</x>
      <y>2</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(225, 225, 225);</string>
    </property>
    <property name="text">
     <string>Xóa đồ thị</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../image.qrc"/>
 </resources>
 <connections/>
</ui>
